### DISCUSSION TITLE
Any ideas why probe timeout and interval is not registering?

### DISCUSSION THREAD

**Discussion by alininja** [litmuschaos/litmus#4539] [Q&A] [answered]
Hi Folks 👋 

Sorry, for the newbie quesiton.

I am trying to get my first experiment working and found this error in the logs of the runner pod:
```
I0318 18:59:56.973969 1 request.go:621] Throttling request took 1.181547645s, request: GET:[https://172.30.0.1:443/apis/apiserver.openshift.io/v1?timeout=32s](https://172.30.0.1/apis/apiserver.openshift.io/v1?timeout=32s)
32024/03/18 18:59:57 Error Creating Resource : ChaosEngine.litmuschaos.io "disk-fill-3nbqx5cv" is invalid: [spec.experiments[0].spec.probe[0].runProperties.interval: Invalid value: "string": spec.experiments[0].spec.probe[0].runProperties.interval in body must be of type integer: "string", spec.experiments[0].spec.probe[0].runProperties.probeTimeout: Invalid value: "string": spec.experiments[0].spec.probe[0].runProperties.probeTimeout in body must be of type integer: "string"]
```

However, the probe in the experiment has `probeTimeout` and `interval` configured, so I am at a lost on how to troubleshoot this.
<img width="1618" alt="Litmus Probe_Settings" src="https://github.com/litmuschaos/litmus/assets/1376738/466d5a13-4aaf-4074-bd74-c5dee77ae262">

Any ideas or guidance will be greatly appreciated.

Thank you for your time 🙏

**Comment by S-ayanide** [comment, marked as answer]
Hey @alininja thanks for sharing this, I see that you have already configured the probe with timeout and interval (in string) and its also getting sent to the subscriber with the correct value, however, according to the error message above it seems that you might have an older CRD since its checking for type `integer` but its had been updated to type `string` with 3.0 and above.

Could you try updating your litmus CRD from here: https://github.com/litmuschaos/litmus/blob/master/chaoscenter/manifests/litmus-portal-crds.yml

And try again!

**Reply by alininja** [reply]
Thank you @S-ayanide for to tip.  Updating to version 3.0.0.beta12 worked.  Thanks again for your help :pray:

**Comment by alininja** [comment]
is there a way to close this discussion without adding another comment?
